Patterns/English/English-Numbers.yaml

```diff
@@ -27,7 +27,7 @@ PlaceHolderPureNumber: !simpleRegex
 PlaceHolderDefault: !simpleRegex
   def: \D|\b
 NumbersWithPlaceHolder: !paramsRegex
-  def: (((?<!\d+\s*)-\s*)|(?<=\b))\d+(?!(\.\d+[a-zA-Z]))(?={placeholder})
+  def: (?<!:\s*)(((?<!\d+\s*)-\s*)|(?<=\b))\d+(?!(\.\d+[a-zA-Z]))(?={placeholder})

```

Comment:
Shouldn't this change be applied to the units package? "00" is a number.
Also, the fix should affect all languages, not only English.